src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -786,6 +784,18 @@ namespace LowTempRadiantSystem {
             }
 
             thisRadSys.CircLength = Numbers(16);
+            
+            thisRadSys.schedNameChangeoverDelay = Alphas(18);
+            if (lAlphaBlanks(18)) {
+                thisRadSys.schedPtrChangeoverDelay = 0;
```

Comment:
Probably could've just relied on initializing the member variable to zero and not setting it to zero here, but it's OK.

Reply:
You are correct--could've been done as you suggested.  Still working out those F90 tendencies.  I can fix this if there are other things to change.

Reply:
Well, still "working it out" is right.  I actually already initialized it in the .hh. 😂 So, I had a hybrid.  Progress, right?  I'll get rid of the unnecessary assignment in line 790.